test/elm/clinical/clinical-test.ts

```diff
@@ -140,24 +175,18 @@ describe('InValueSet', () => {
     (await this.nullCode.exec(this.ctx)).should.be.false();
   });
 
```

Comment:
```suggestion

  it('should return false if the passed in code is null', async function () {
    (await this.nullCastAsCode.exec(this.ctx)).should.be.false();
  });

  it('should return false if the passed in code is null and the valueset is null', async function () {
    (await this.nullCastAsCodeWithNullVS.exec(this.ctx)).should.be.false();
  });

```

Tests for the `null as Code` and `null as ValueSet` scenarios.

Reply:
added in [63a8e73](https://github.com/cqframework/cql-execution/pull/341/commits/63a8e73a4de4ebcf675127bca4af4cba03928af7)